freestyle/shared/src/main/scala/freestyle/module.scala

```diff
@@ -52,9 +50,7 @@ object openUnion {
       // cs is a `@module` is the _class_ of the `object` extends from `FreeModuleLike`
       // Note that a trait's companion object has a _real_ class, unknown, to rule its behaviour.
       def isModuleFS(cs: ClassSymbol): Boolean =
-        cs.companion.asModule.moduleClass.asClass.baseClasses.exists {
-          case x: ClassSymbol => x.name == TypeName("FreeModuleLike")
-        }
+        cs.baseClasses.exists( _.name == TypeName("FreeModuleLike") )
```

Comment:
Yes and no... thing is that the `cs` is the `ClassSymbol` that points to the `@module`-annotated trait. What the remove line was doing was "go to the companion object and see if its _class_ inherits from `FreeModuleLike`". Note that _its class_ is the one that defines the behaviour and operation of the object, but it is not the one that the companion object is supposed to accompany.